File: lab02/zadanie.js
const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

const {NodeSSH}=require('node-ssh');

const ssh= new NodeSSH();
let ścieżka = ''

rl.question("Podaj ścieżkę", function(ścieżkaPodana) {
        ścieżka = ścieżkaPodana
        rl.close();
});

ssh.connect({
    host: 'sigma.ug.edu.pl',
    username:'dpionk',
    privateKey: 'C:/Users/Darusia/.ssh/id_rsa'
}).then(() => {
    ssh.execCommand(`cat ${ścieżka}`).then((result) => {
        console.log(result.stdout);
    });
}).catch(err => console.log(err))

File: lab02/zadanie.py
import paramiko
ssh = paramiko.SSHClient()
file = input('Podaj nazwę pliku')

ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())

ssh.connect('sigma.ug.edu.pl', username='dpionk', key_filename='~/.ssh/id_rsa')

stdin, stdout, stderr = ssh.exec_command("cat ${file}")
ssh.close()